docs/jdbct4ref_guide/pom.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+ <!-- 
+* @@@ START COPYRIGHT @@@                                                       
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+-->
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.trafodion</groupId>
+  <artifactId>jdbct4-ref-guide</artifactId>
+  <version>${env.TRAFODION_VER}</version>
+  <packaging>pom</packaging>
+  <name>Trafodion JDBC Type 4 Programmer's Reference Guide</name>
+  <description>This document describes how to use the JDBC Type 4 Driver.</description>
+  <url>http://trafodion.incubator.apache.org</url>
+  <inceptionYear>2015</inceptionYear>
+
+  <parent>
+    <groupId>org.apache.trafodion</groupId>
+    <artifactId>trafodion</artifactId>
+    <version>1.3.0</version>
```

Comment:
Document versions are driven from env.TRAFODION_VERSION. The value here is
required but doesn't have a meaning that I'm aware of.

On Fri, Apr 29, 2016 at 5:27 PM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/jdbct4ref_guide/pom.xml
> <https://github.com/apache/incubator-trafodion/pull/457#discussion_r61657384>
> :
>
> > +* @@@ END COPYRIGHT @@@
> > +-->
> > +  <modelVersion>4.0.0</modelVersion>
> > +  <groupId>org.apache.trafodion</groupId>
> > +  <artifactId>jdbct4-ref-guide</artifactId>
> > +  <version>${env.TRAFODION_VER}</version>
> > +  <packaging>pom</packaging>
> > +  <name>Trafodion JDBC Type 4 Programmer's Reference Guide</name>
> > +  <description>This document describes how to use the JDBC Type 4 Driver.</description>
> > +  <url>http://trafodion.incubator.apache.org</url>
> > +  <inceptionYear>2015</inceptionYear>
> > +
> > +  <parent>
> > +    <groupId>org.apache.trafodion</groupId>
> > +    <artifactId>trafodion</artifactId>
> > +    <version>1.3.0</version>
>
> So, this is the 1.3.0 version? Is there also a 2.0,0 version?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/457/files/a3cc57d7f87c8f89148fe34f729e9c630e9e2091#r61657384>
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*